src/locales/ko/mystery-encounters/fiery-fallout-dialogue.json

```
@@ -1,26 +1,26 @@
 {
-  "intro": "You encounter a blistering storm of smoke and ash!",
-  "title": "Fiery Fallout",
-  "description": "The whirling ash and embers have cut visibility to nearly zero. It seems like there might be some... source that is causing these conditions. But what could be behind a phenomenon of this magnitude?",
-  "query": "What will you do?",
+  "intro": "매캐한 연기와 화산재의 폭풍을 맞닥뜨렸다!",
```

```suggestion
  "intro": "매캐한 연기와 화산재 폭풍을 맞닥뜨렸다!",
```
'의'를 빼는 편이 조금 자연스럽다고 생각합니다.